### PR TITLE
Add invoice's pdf and hosted url

### PR DESCRIPTION
**Pull request by remi-stripe**
r? @brandur-stripe 
cc @stripe/api-libraries

### CODE REVIEW

File: invoice.go [hunk 1]
```diff
@@ -90,6 +90,8 @@ type Invoice struct {
 	DueDate             int64             `json:"due_date"`
 	EndingBalance       int64             `json:"ending_balance"`
 	Forgiven            bool              `json:"forgiven"`
+	HostedInvoiceURL    string            `json:"hosted_invoice_url"`
+	InvoicePDF          string            `json:"invoice_pdf"`
```
Comment (brandur-stripe) [inline invoice.go]: Sorry for the nit, but I think this should be below `ID` for it to line up alphabetically.

**Comment by brandur-stripe** [comment]
Thanks Remi!

**Comment by brandur-stripe** [comment]
Just a quick bump on this one — I suspect the notification may have been accidentally archived in your inbox during the churn today.

**Comment by remi-stripe** [comment]
ah yes, totally missed this one. The other two libraries (dotnet and java) put id/object at the top I think so I missed this.
PTAL.

**Comment by brandur-stripe** [comment]
Released as 32.2.0.

**Pull request merged**